data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			has "Hai Witness Unfettered [1]"
+			has "ZZ Hai Tour [3] Meet Unfettered: done"
+	on offer
+		conversation
+			`You had to bribe your way through just to land without being blown up. These Unfettered Hai do not seem friendly, but it's time to find out what they're about.`
+				accept
+	on complete
+		conversation
+			`You dodge the conflict in space and come in to land. Cloudfire's atmosphere is unusually hazy, with almost constant cloud cover, making agriculture unproductive here even though it is a warm and damp environment. There are several major cities here that were abandoned centuries ago, due to conflict with the Unfettered. The one major city that remains is the closest thing the Hai have to a military base, with new supplies of weapons and ammunition constantly pouring in from the rest of Hai space.`
+			`	The spaceport has massive hangars and advanced repair facilities designed to accommodate the largest Hai warships. Where a fleet has just returned from a recent battle, the hangars are lit with flashes from robotic welders as damaged hull plates are swapped out for fresh ones and weakened joints are reinforced. The actual process of building these warships, however, takes place on the Hai homeworld, where the facilities will not be vulnerable to attack.`
+			`	It does not take long to realise that this conflict really is constant, even if the friendlier Hai on this side seem to be maintaining their defence without apparently taxing their civilisation to any great extent.`
+
+# Both tours merge here
+mission "Hai Tour [5] Tour Ends"
+	landing
+	name "Head directly to Hai-home"
+	description "Go to Hai-home and meet whoever wants to speak with you."
+	source "Cloudfire"
+	destination "Hai-home"
+	clearance
+	to offer
+		has "00 Hai Tour [4] Cloudfire: done"
+	on offer
+		conversation
+			branch "skipped tours"
+				has "Hai Witness Unfettered [1]"
+			`After looking around Cloudfire for some time you stop in the spaceport for a moment to absorb everything you've seen. A screen is playing through news on the wall nearby and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+				goto "that's awful"
+			label "skipped tours"
+			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall, "Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
```

Comment:
```suggestion
			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall: "Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
```
You could also use a period instead of a semicolon after `wall`, but the comma needs to be changed to one of those two.